### PR TITLE
config: add personorg identifiers schemes

### PR DESCRIPTION
**Pull request by kpsherva**
* closes https://github.com/CERNDocumentServer/cds-rdm/issues/226

### CODE REVIEW

File: invenio.cfg [hunk 1]
```diff
@@ -445,3 +445,8 @@ RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
                                       "inspire": {"label": _("Inspire"),
                                                   "validator": schemes.is_inspire,
                                                   "datacite": "INSPIRE"}}}
+
+RDM_RECORDS_PERSONORG_SCHEMES = {**RDM_RECORDS_PERSONORG_SCHEMES,
+                                 **{"inspire": {"label": _("Inspire"),
```
Comment (ntarocco) [inline invenio.cfg:450]: I think the name is capital letters:
```suggestion
                                 **{"inspire": {"label": _("INSPIRE"),
```

File: invenio.cfg [hunk 2]
```diff
@@ -445,3 +445,8 @@ RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
...
+                                                "validator": schemes.is_inspire,
```
Comment (ntarocco) [inline invenio.cfg:451]: I doubt that there is any detection in place?

**Review by ntarocco — APPROVED** [review]

File: invenio.cfg (hunk 2 above)
Comment (kpsherva) [reply invenio.cfg:451]: this is validation, not detection. Basically it checks if the ID is a number, otherwise it fails

**Review by jrcastro2 — APPROVED** [review]
LGTU! Peer reviewed with Fatimah.

**Pull request merged**